packages/engine-test-utils/src/__tests__/pods-core/MarkdownPod.spec.ts

```diff
@@ -58,6 +79,46 @@ describe("markdown publish pod", () => {
       { expect, preSetupHook: ENGINE_HOOKS.setupRefs }
     );
   });
+
+  test("wikiLinktoURL with siteUrl", async () => {
```

Comment:
can you add additional test cases here?

- test with xvault link to same vault
- test with xvault link to a seed vault with different url
- test with a regular link in a seed vault that has a different url

See [here](https://github.com/dendronhq/dendron/blob/master/packages/engine-test-utils/src/__tests__/dendron-cli/commands/seedCLICommand.spec.ts#L1:L1) for an example about testing with seeds